code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -940,6 +940,20 @@ datum/reagent/medicine/styptic_powder/overdose_start(mob/living/M)
 	description = "Reacts with neural tissue, helping reform damaged connections. Can cure minor traumas."
 	color = "#EEFF8F"
 
+/datum/reagent/neurine/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
+	var/obj/item/organ/brain/B = M.getorganslot(ORGAN_SLOT_BRAIN)
+	if(!B)
```

Comment:
```
	if(!B || !CHECK_BITFIELD(B.organ_flags, ORGAN_FAILING))
```
or
```
	if(!B || !(B.organ_flags & ORGAN_FAILING))
```

Reply:
I always have trouble getting this to work as expected, but I'm gonna trust the ghom.